src/uct/tcp/tcp.h

```diff
@@ -193,10 +193,10 @@ enum {
  * TCP connection request packet
  */
 typedef struct uct_tcp_cm_conn_req_pkt {
-    uct_tcp_cm_conn_event_t       event;      /* Connection event ID */
-    uint8_t                       flags;      /* Packet flags */
-    struct sockaddr_in            iface_addr; /* Socket address of UCT local iface */
-    uct_tcp_ep_cm_id_t            cm_id;      /* EP connection mananger ID */
+    uct_tcp_cm_conn_event_t event;      /* Connection event ID */
+    uint8_t                 flags;      /* Packet flags */
+    struct sockaddr_storage iface_addr; /* Socket address of UCT local iface */
```

Comment:
this is increasing packet size, better move to packet end (so it could be according to the real address size)